ftw/linkchecker/command/checking_links.py

```diff
@@ -0,0 +1,35 @@
+from zope.component.hooks import setSite
+from Testing.makerequest import makerequest
+from AccessControl.SecurityManagement import newSecurityManager
+from zope.globalrequest import setRequest
+import AccessControl
+from zc.relation.interfaces import ICatalog
+from zope.component import getUtility
+
+
+def main(plone, *args):
+
+    # Set up request for debug / bin/instance run mode.
+    app = makerequest(plone)
+    setRequest(app.REQUEST)
+
+    houptding = app.restrictedTraverse('/')
```

Comment:
Uhm, what? No. Use sensible english variable names, like `app = `. But this isn't needed, since `app` is already set up just above on line 13.

Reply:
👍